### ISSUE TITLE
Loop requests to json file

### ISSUE THREAD

**Issue by irongomme** [globocom/hlsclient#48]
I've noticed that the json stream config is requesting continually, is there a reason ?

**Comment by jbochi** [comment]
Yes. If you generate the config file dynamically, hlsclient will handle new streams automatically.

**Comment by irongomme** [comment]
this is a good Thing,  but several requests by second is a waste of bandwidth , is there a way to regulate this ?
